packages/sdk-components-react/src/props.ts

```diff
@@ -30,3 +30,8 @@ export { propsMeta as Vimeo } from "./vimeo.ws";
 export { propsMeta as VimeoPreviewImage } from "./vimeo-preview-image.ws";
 export { propsMeta as VimeoPlayButton } from "./vimeo-play-button.ws";
 export { propsMeta as VimeoSpinner } from "./vimeo-spinner.ws";
+export {
+  propsMetaTooltip as Tooltip,
+  propsMetaTooltipContent as TooltipContent,
+  propsMetaTooltipTrigger as TooltipTrigger,
+} from "./radix-tooltip.ws";
```

Comment:
Radix components should have a separate package

Reply:
I agree, ideally yes, is it easy to split now? the lets split, if not - lets wait until we build the "install package ui"